api/src/services/messaging.js

```diff
@@ -60,7 +60,7 @@ const applyTaskStateChangesToDocs = (taskStateChanges, docs) => {
 };
 
 const getOutgoingMessageService = () => {
-  const settings = config.get('sms');
+  const settings = config.get('sms') || { sms: { outgoing_service: 'medic-gateway' } };
```

Comment:
I think we also need to add the default value to the `isMedicGatewayEnabled` function from the same service. 
The method is called by `sms-gateway` controller when passing outgoing messages to `/api/sms`. Without it, it always returns an empty list.

Reply:
My search and replace skills failed me... Fixed!

Reply:
Also, the default object is wrong - it should just be `{ outgoing_service: 'medic-gateway' }`. SMH.